installer/cluster.go

```diff
@@ -578,8 +578,12 @@ iptables -A FORWARD -i eth0 -j DROP
 
 var startScript = template.Must(template.New("start.sh").Parse(`
 #!/bin/sh
+set -e -x
+
 FIRST_BOOT="/var/lib/flynn/first-boot"
-if [[ ! -f "${FIRST_BOOT}" ]]; then
+mkdir -p /var/lib/flynn
+
+if [ ! -f "${FIRST_BOOT}" ]; then
```

Comment:
Any reason we aren't using `/bin/bash`? That is the change I would have made rather than this, but I don't think it *really* matters.